src/clr-angular/forms/select/select.ts

```diff
@@ -18,6 +19,7 @@ import {TAB} from "../../utils/key-codes/key-codes";
 })
 export class ClrSelect implements AfterContentInit {
     @ViewChild("input") input: ElementRef;
+    @ContentChild(ClrOptions) options: ElementRef;
```

Comment:
The type here is wrong. Either you want:
```typescript
@ContentChild(ClrOptions) options: ClrOptions;
```
or 
```typescript
@ContentChild(ClrOptions, {read: ElementRef}) options: ElementRef;
```
I believe in your case the first one is simpler and will work fine. But my guess is that the current typing would throw an error when compiled AoT. And if it doesn't, it should. 😛

Reply:
fixed